index.bs

```diff
@@ -5830,6 +5831,78 @@ At this time, one [=credential property=] is defined: the [=resident key credent
 :: None.
 
 
+## Large blob storage extension (<dfn>largeBlob</dfn>) ## {#sctn-large-blob-extension}
+
+This [=client extension|client=] [=authentication extension=] allows a [=[RP]=] to store opaque data associated with a credential. Since [=authenticators=] can only store small amounts of data, and most [=[RPS]=] are online services that can store arbitrary amounts of state for a user, this is only useful in specific cases. For example, the [=[RP]=] might wish to issue certificates rather than run a centralised authentication service.
+
+Note: [=[RPS]=] can assume that the opaque data will be compressed when being written to a space-limited device and so need not compress it themselves.
+
+Since a certificate system needs to sign over the public key of the credential, and that public key is only available after creation, this extension is not defined in the [=registration extension|registration=] context.
+
+Since certificates are sizable relative to the storage capabilities of typical authenticators, user agents SHOULD consider what indications and confirmations are suitable to best guide the user in allocating this limited resource and prevent abuse.
+
+Note: In order to interoperate, user agents storing large blobs on authenticators using [[!FIDO-CTAP]] are expected to use the provisions detailed in that specification for storing [=large, per-credential blobs=].
```

Comment:
The last bit currently links to https://fidoalliance.org/specs/fido-v2.0-ps-20190130/fido-client-to-authenticator-protocol-v2.0-ps-20190130.html#large-blob on the editor's draft, which has no information on this extension?

Reply:
Correct, it's prospective:

> FIDO will need to rev the public version of CTAP2, and WebAuthn will have to update the base URL, for it to resolve correctly.

Reply:
It should be pointing to CTAP 2.1.   Unfortunately the current public Review draft is also a bit old.
https://fidoalliance.org/specs/fido2/fido-client-to-authenticator-protocol-v2.1-rd-20191217.html

The goal is to have a new public RD out shortly.

Reply:
@agl missed that part - thanks!